src/spring-mongo/src/main/java/net/conology/spring/restjsonpath/mongo/NestedValueTestCompiler.java

```diff
@@ -54,15 +52,12 @@ public MongoPropertyCondition compile() {
     }
 
     private MongoPropertyCondition createTest(MongoFieldSelector fieldSelector, MongoElementMatch elementMatch) {
-        var assertion =
-            elementMatch != null ? elementMatch
-            : ( finalAssertion != null ? finalAssertion
-                : parent.getExistenceAssertion()
-            );
+        var assertion = elementMatch != null ? elementMatch
+                : (finalAssertion != null ? finalAssertion
+                        : parent.getExistenceAssertion());
         return new MongoPropertyCondition(
-            fieldSelector,
-            assertion
-        );
+                fieldSelector,
+                assertion);
```

Comment:
this is my personal ikes. but again, i'd jsut avoid reformatting lines of code i didn't work on